.concourse/post-publish.yaml

```diff
@@ -0,0 +1,346 @@
+
+resource_types:
+- name: pull-request
+  type: docker-image
+  source:
+    repository: teliaoss/github-pr-resource
+- name: github-status
+  type: docker-image
+  source:
+    repository: resource/github-status
+    tag: release
+resources:
+  - name: kubecf
+    type: git
+    source:
+      branch: master
```

Comment:
This should be a dynamic value.

Reply:
Yup, same as https://github.com/SUSE/kubecf/pull/360/files#r371082388 . We will need to template the pipeline when we want to expand it to other branches